src/components/session-submission/session-submission.tsx

```diff
@@ -0,0 +1,425 @@
+import React, { Component, ReactNode } from 'react';
+import { graphql, StaticQuery } from 'gatsby';
+import { Session } from './session.model';
+import { Presenter } from './presenter.model';
+import { SessionLevels } from './session-level.enum';
+import { OpenLabInterests } from './open-lab-interest.enum';
+import { Formik, FieldArray, Field, ArrayHelpers, ErrorMessage, FormikErrors, FormikTouched, getIn } from 'formik';
+import { sessionSchema } from './session-schema';
+import { WithContext as ReactTags } from 'react-tag-input';
+import base from './../../services/firebase';
+import { AuthService } from './../../services/authentication'
+
+const KeyCodes = {
+    comma: 188,
+    enter: 13,
+};
+
+const delimiters = [KeyCodes.comma, KeyCodes.enter];
+type SessionSubmissionState = {
+    session: Session;
+    validSubmission: boolean;
+    submissionError?: string;
+};
+
+export default class SessionSubmission extends Component {
+
+    public state: SessionSubmissionState = {
+        session: this.buildEmptySession(),
+        validSubmission: false,
+        submissionError: undefined
+    };
+
+    public render(): ReactNode {
+        return (
+            <StaticQuery
+                query={ submitSessionQuery }
+                render={ ({ content }) => this.renderForm(content) } />
+        );
+    }
+
+    public renderForm({ title, termsAndConditionsAgreement, submissionConfirmation }: any): ReactNode {
+        return (
+            <div className="container">
+                {
+                    this.state.validSubmission ?
+                        <div className="row justify-content-center">
+                            <div className="col-12 col-md-10 mt-5" dangerouslySetInnerHTML={{ __html: submissionConfirmation.childContentfulRichText.html }} />
+                        </div> :
+                        <>
+                            <h1>{ title }</h1>
+                            <Formik
+                                initialValues={this.state.session}
+                                validationSchema={sessionSchema}
+                                onSubmit={async (values, { setSubmitting }) => {
+                                    try {
+                                        await base.post(
+                                            `2019/${AuthService.getUser().userId}/submitted-sessions/${values.title}`, {
+                                                data: values
+                                            });
+                                        this.setState({
+                                            ...this.state,
+                                            validSubmission: true
+                                        });
+                                    } catch (error) {
+                                        this.setState({
```

Comment:
add console.error here for quicker debugging

Reply:
lint rules say
> REEEEEEEEE

Reply:
ignore 'em rip